### PR TITLE
Variants are escaped with slashes

### PR DESCRIPTION
**Pull request by ericclemmons**
There appear to be some added `\` characters for the variants being generated.

> Screen Shot 2019-05-20 at 12 48 40 AM

### CODE REVIEW

File: src/config/theme.js [hunk 1]
```diff
@@ -276,32 +276,32 @@ export const cursor = {
 export const width = {
   auto: "auto",
   ...spacing,
-  "1\\/2": "50%",
```
Comment (ericclemmons) [inline src/config/theme.js]: This is largely a revert of 48352680fb10219be7b98c65a1d367b25fd408da

File: src/config/createVariant.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ export default function createVariant(utilities, generateDeclarations, prefix) {
     return {}
   }
   return Object.keys(utilities).reduce((variants, utilityKey) => {
-    variants[`${prefix}\\:${utilityKey}`] = generateDeclarations(
+    variants[`${prefix}:${utilityKey}`] = generateDeclarations(
```
Comment (ericclemmons) [inline src/config/createVariant.js]: Revert of 5e472a74716a23338a9519412c90f7d2137dd914

File: scripts/css.js [hunk 3]
```diff
@@ -31,7 +31,10 @@ const utilities = Object.keys(benefit.utilities)
     const inserted = emotion.cache.inserted[hash]
 
     // Replae `css-1234` with `hover:opacity-0`
-    return inserted.replace(emotionclassName, utility)
+    return inserted.replace(
+      emotionclassName,
+      utility.replace(":", "\\:").replace("/", "\\/")
```
Comment (ericclemmons) [inline scripts/css.js]: The _real_ fix!

**Comment by cdonohue** [comment]
Thanks @ericclemmons!

**Pull request merged**